### DISCUSSION TITLE
Cannot see the code on github after pushing the code from VS code

### DISCUSSION THREAD

**Discussion by josebGithub** [community/community#43245] [Actions]
### Select Topic Area

Product Feedback

### Body

I used Visual Studio code to submit my code to github and push the code there. Message told me that everything is up-to-date. But I cannot see the submit code in github even I re-initialize it and submit the code and push the code again?

Message from git
============= 
git push -u origin main
branch 'main' set up to track 'origin/main'.
Everything up-to-date

All I see on github is the page below:

enter image description here
![github](https://user-images.githubusercontent.com/46608139/210438289-dc2556a5-b8da-4d17-8817-ea109d6e8d36.jpg)

**Comment by nicholasbergesen** [comment]
Hi @josebGithub,

Could you please run this git command to confirm that your local git checkout it pointing to your remote github repository
`git remote -v`

If the remote repository url doesn't look correct, you can follow [this](https://docs.github.com/en/get-started/getting-started-with-git/managing-remote-repositories#changing-a-remote-repositorys-url) documentation to update it to point to your github repository.

**Reply by josebGithub** [reply]
Yes, I found this command yesterday and I solved the problem. I removed the old remote local one and create a new remote locate one and it works fine now. Thanks anyway!

**Comment by bitflame** [comment]
nick's advice worked like a charm for me. Thank you.

**Comment by arsheenkour** [comment]
mine file is visible but the code inside it isnt

**Reply by anjaliirathorr** [reply]
How did you solve this problem. Please help

**Comment by arsheenkour** [comment]
While you write the code on VS code  just save it and then go with the
procedure of pushing it onto GitHub.
I hope it works for you.
